platforms/hyperledger-besu/configuration/roles/create/helm_component/templates/crypto_tessera.tpl

```diff
@@ -28,7 +28,7 @@ spec:
       # alpineutils image has the binaries like jq,curl, wget and openssl in it. 
       # having this image with binaries helps in removing the dependencies of utility binaries in the besu and tessera containers
       # dockerfile can be found at shared/images/apline-utils.Dockerfile
-      alpineutils: {{ network.docker.url }}/alpine-utils:1.1
+      alpineutils: "{{ network.docker.url }}/alpine-utils:{{ network.bevel_version | default('latest') }}"
```

Comment:
This should not be changed as `network.bevel_version` gives a wrong sense of information that by changing to lets say, `1.0` version, will indeed make use of Bevel 1.0 code.
Bevel versions are heavily affected by code changes and the right way to use a specific version of Bevel is via changing the `tag` of the local Bevel repo (from where the deployments are made).